modules/rodoo-enable-override.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/run_once_duration_override/run-once-duration-override-install.adoc
+
+:_content-type: PROCEDURE
+[id="rodoo-enable-override_{context}"]
+= Enabling the run-once duration override on a namespace
+
+To apply the run-once duration override from the {run-once-operator} to run-once pods, you must enable it on each applicable namespace.
+
+.Prerequisites
+
+* You have installed the {run-once-operator}.
+* You have access to the cluster with `cluster-admin` privileges.
++
+// TODO: are cluster-admin privileges required here to label? what if it's their own namespace?
```

Comment:
@ingvagabund For most of the procedures in the docs, we say that you need cluster-admin privileges. I'm assuming you need to be a cluster-admin to label a namespace too, but wanted to double check.

Reply:
Good question. As a user you can only label namespaces that you created. So practically you could remove a label from a namespace to avoid overriding of `activeDeadlineSeconds` fields. There's unfortunately no protection for it.

Otherwise, you don't need to have cluster-admin privileges in general. Admins are the target audience for installing the operator. They need to make sure all affected labels are kept labeled. Which is outside of scope of the operator.

Reply:
Okay thanks, I'll remove the requirement for cluster-admin then